WcaOnRails/app/views/competitions_mailer/notify_wcat_of_confirmed_competition.html.erb

```diff
@@ -26,6 +26,7 @@
   <% if Team.probation.current_members.map { |member| member.user_id }.include?(delegate.id) %>
     <p class="alert">
       <%= delegate.name %> is currently on probation. Please check the <%= link_to "Delegates on Probation spreadsheet", "https://docs.google.com/spreadsheets/d/1ZU1lk0niGn-vjmbiDkoROU8qjmdHkNebZ2GCFHm6vvA/edit#gid=1887522006" %> for more details.
+      You can view the Delegate's currently announced competitions <%= link_to "here", "https://www.worldcubeassociation.org/competitions?delegate=" + delegate.id.to_s %>.
```

Comment:
Does `competitions_url(delegate: delegate.id)` work here instead? (these url helpers are nice because they'll do the right thing in other environments like local and staging)

Reply:
Perfect! I was looking around for something like this with no success.

Where is competitions_url defined?

Reply:
It comes from `Rails.application.routes.url_helpers`, which is autogenerated from our routes.rb file. If you ever want to see more information about the available url helpers, you can run `bin/rake routes` locally, or browse to http://localhost:3000/rails/info/routes locally.